students/dychowicz_monika/lesson_07/TheNumberOfWords

```diff
@@ -0,0 +1,7 @@
+def main():
+    text = input()
+    print(1 + (text.count(' ')))
```

Comment:
this won't works.
for example for `input ='one          two '` output should be 2, where it is now 12